frontend/OBSApp.cpp

```diff
@@ -15,108 +15,62 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ******************************************************************************/
 
-#include <time.h>
-#include <stdio.h>
-#include <wchar.h>
-#include <chrono>
-#include <ratio>
-#include <string>
-#include <sstream>
-#include <mutex>
-#include <filesystem>
-#include <util/bmem.h>
-#include <util/dstr.hpp>
-#include <util/platform.h>
-#include <util/profiler.hpp>
-#include <util/cf-parser.h>
-#include <obs-config.h>
-#include <obs.hpp>
-#include <qt-wrappers.hpp>
-#include <slider-ignorewheel.hpp>
-
-#include <QDir>
-#include <QFile>
-#include <QGuiApplication>
-#include <QScreen>
-#include <QProcess>
-#include <QAccessible>
-
-#include "obs-app.hpp"
-#include "obs-proxy-style.hpp"
-#include "log-viewer.hpp"
-#include "volume-control.hpp"
-#include "window-basic-main.hpp"
-#ifdef __APPLE__
-#include "window-permissions.hpp"
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
 #endif
-#include "window-basic-settings.hpp"
-#include "platform.hpp"
 
-#include <fstream>
+#include "OBSApp.hpp"
 
-#include <curl/curl.h>
+#include <components/Multiview.hpp>
+#include <utility/OBSEventFilter.hpp>
+#if defined(_WIN32) || defined(ENABLE_SPARKLE_UPDATER)
+#include <utility/models/branches.hpp>
```

Comment:
What was the rationale for changing this order?

Reply:
Case sensitive sorting was applied, which puts the model import below the event filter import.